src/env.h

```diff
@@ -1101,7 +1103,8 @@ class Environment : public MemoryRetainer {
   uint32_t script_id_counter_ = 0;
   uint32_t function_id_counter_ = 0;
 
-  AliasedUint32Array exiting_;
+  enum ExitInfoFields { kExiting = 0, kExitCode, kHasExitCode };
```

Comment:
I used plain enum to declare index identifiers here instead of using enum class since kExiting, kExitCode, and kHasExitCode seem unique enough in `Environment`. Not sure if that looks good.